package.json

```diff
@@ -7,8 +7,13 @@
     "cheerio": "^0.22.0",
     "node-fetch": "~2.2.0"
   },
-  "devDependencies": {},
-  "scripts": {},
+  "devDependencies": {
+    "@types/jest": "^25.1.2",
```

Comment:
Since neither the library nor the tests use TypeScript, can you remove the dev dependency on the jest typings?

Reply:
Yes, removed the dependency for `@types/jest`